### DISCUSSION TITLE
How to use withset in Conditionals

### DISCUSSION THREAD

**Discussion by quekyj** [intel/rohd#363] [Q&A]
How to do assignment to a subset of bus inside conditionals? 

Example, inside always block we want to do something like this:

```dart
shift_register[2:1] <= shift_register[1:0];
shift_register[0] <= shift_register[2] ^ shift_register[1];
```

**Comment by mkorbel1** [comment]
This capability does not currently exist, but it could be achievable.  Nice suggestion!

Filed an issue tracking this here: https://github.com/intel/rohd/issues/378
